src/main/java/com/gl/donate_receive/service/converter/ItemConverter.java

```diff
@@ -6,21 +6,24 @@
 import com.gl.donate_receive.model.ItemType;
 import org.springframework.stereotype.Component;
 
+import java.util.UUID;
+
 @Component
 public class ItemConverter {
 
-	public Item dtoToModel(ItemDto itemDto) {
+	public Item dtoToModel(ItemDto itemDto, UUID ownerId) {
 		return Item.builder()
 			.name(itemDto.getName())
 			.description(itemDto.getDescription())
 			.type(itemDto.getType())
 			.status(getStatus(itemDto))
+			.ownerId(ownerId)
 			.build();
 	}
 
 	private ItemStatus getStatus(ItemDto itemDto) {
 		return itemDto.getType().equals(ItemType.AVAILABLE) ?
 			ItemStatus.RECEIVABLE : ItemStatus.DONATABLE;
```

Comment:
This looks odd. 
![image](https://user-images.githubusercontent.com/108729712/177929909-03078131-c48a-4dd6-a76d-c4fd04ba5961.png)

Reply:
In my mind schema looks like
![image_2022-07-08_09-41-43](https://user-images.githubusercontent.com/42956163/177932640-69b45cc1-ed5c-4e41-9494-3d45694f2fbc.png)
: